public/index.html

```diff
@@ -7,6 +7,10 @@
             name="description"
             content="Seamless, cheap and fast L2 token swapping built on Ethereum. Join the fast lane."
         />
+        <link
+            href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600;700&display=swap"
+            rel="stylesheet"
+        />
```

Comment:
@mfinestone @luzzif 

Using fonts.googleapis.com doesn't work in China. It's the reason that we would like to not load the fonts from a CDN

Reply:
From our discussion, I will approve the PR. Users in China won't get the fonts.

Reply:
Oh, sorry, didn't know that! I'm going to submit a second PR with locally hosted fonts.